### ISSUE TITLE
6.3.0 SPM support from master

### ISSUE THREAD

**Issue by apps4everyone** [apache/cordova-ios#1316]
Hey everyone,

I am pleased and super happy that there is a new release but it look's like 6.3.0 was not branched from master.
Therefore the fixes are only in 6.3.0 but without the SPM support from master

Currently there are two options:
1.  6.3.0 with no SPM support
2. Master without 6.3.0 fixes

Would it be possible to merge the 6.3.0 fixes also to master?

Thank you
Stefan

**Comment by dpogue** [comment]
All of the fixes included in 6.3.0 were first merged to the master branch and then backported, so there shouldn't be anything missing.

**Comment by apps4everyone** [comment]
ok, strange, couldn't load the SPM. Will retry. thx for the very quick info.

**Comment by apps4everyone** [comment]
it look's like the commits from NiklasMerz are only in 6.3.0
https://github.com/apache/cordova-ios/commits/rel/6.3.0
https://github.com/apache/cordova-ios/commits?author=NiklasMerz
can't see the changes e.g. 
https://github.com/apache/cordova-ios/commit/6cfcbf8c74b1df37d6deee1fb92eebfd82dcdd7d
in master
🤔

**Comment by dpogue** [comment]
https://github.com/apache/cordova-ios/commit/f804a42e171b2ec9288421e0ce63eb35bb0afe14 is where that was merged to master.

I see that code on the master branch here: https://github.com/apache/cordova-ios/blob/master/CordovaLib/Classes/Private/Plugins/CDVWebViewEngine/CDVWebViewEngine.m#L405-L410

**Comment by apps4everyone** [comment]
master has still old release notes 6.2.0
https://github.com/apache/cordova-ios/blob/master/RELEASENOTES.md
if everything is first in master why is then the version bump only in 6.3.0?
https://github.com/apache/cordova-ios/blob/rel/6.3.0/RELEASENOTES.md

sorry for my confusing questions

**Comment by apps4everyone** [comment]
what would be the equivalent commit hash for 6.3.0 on master?

**Comment by dpogue** [comment]
There isn't one.

The master branch contains a bunch of work for the next major version release (including breaking API changes to support distributing cordova-ios as a Swift package). We haven't finished work on 7.0.0 yet though.

There were a few high-impact bug fixes that were merged to master, that we decided to backport to a minor release on the existing 6.x branch so that people could benefit from those fixes without needing to wait for 7.0.0 (and without needing to deal with breaking changes).

The currently supported version of cordova-ios that people should use is 6.3.0.
